ShippingApp/target/classes/application.yml

```diff
@@ -0,0 +1,18 @@
+server:
+  port: 8080
+
+config-rabbit:
+  queue: db-queue
+  exchange: db-exchange
+  routing-key: "#"
+
+spring:
+  rabbitmq:
+    host: javaschoolrmqrv2dutokzu7as-vm0.southcentralus.cloudapp.azure.com
+    port: 5672
+    username: user
+    password: JavaSchool.1234
+
+health:
+  rabbit:
+    enabled: false
```

Comment:
Remove the target binary files from git, they shouldn't be pushed

Reply:
They were added to the gitignore.